src/cross_module_fn.c

```diff
@@ -24,6 +26,18 @@ TS_FUNCTION_INFO_V1(ts_finalize_agg_sfunc);
 TS_FUNCTION_INFO_V1(ts_finalize_agg_ffunc);
 TS_FUNCTION_INFO_V1(continuous_agg_invalidation_trigger);
 
+Datum
+ts_set_integer_now_func(PG_FUNCTION_ARGS)
```

Comment:
Should setting the function be apache-licensed?

Reply:
We discussed this with @cevian and decided against it as anything using the set `now_func` would be enterprise.
Also, I think the function had some dependencies in the tsl folder which could complicate making it Apache licensed